src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -67,9 +67,9 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone, email and address, but does not match name
+        // Keywords match phone and email, but does not match name
```

Comment:
good job for also remembering to edit the comment!

Reply:
Thanks for the comment.